test/fixtures/output/connect/plugin/install/remote-specific-version.golden

```diff
@@ -0,0 +1,24 @@
+[DRY RUN] Performing a dry run of this command.
+
+Using the only available Confluent Platform installation at "test/fixtures/input/connect/confluent-7.3.3".
+Do you want to install this plugin into "test/fixtures/input/connect/confluent-7.3.3/share/confluent-hub-components"? (y/n): 
+License:
+Example License 1.0
+url-to-license-goes-here
+I agree to this software license agreement. (y/n): 
+Installing Integration Test Plugin 0.0.5, provided by Confluent, Inc.
+[DRY RUN] Skipping plugin installation.
+
+Detected the following worker configuration files:
+    |                                         Path                                         | Description  
+----+--------------------------------------------------------------------------------------+--------------
+  1 | test/fixtures/input/connect/confluent-7.3.3/etc/kafka/connect-distributed.properties | Standard     
+  2 | test/fixtures/input/connect/confluent-7.3.3/etc/kafka/connect-standalone.properties  | Standard     
+Do you want to update all detected config files? (y/n): 
+Adding plugin installation directory to the plugin path in the following files:
+	* test/fixtures/input/connect/confluent-7.3.3/etc/kafka/connect-distributed.properties
+	* test/fixtures/input/connect/confluent-7.3.3/etc/kafka/connect-standalone.properties
+[DRY RUN] Skipping update of worker configuration file "test/fixtures/input/connect/confluent-7.3.3/etc/kafka/connect-distributed.properties".
```

Comment:
Instead of printing "skipping update", I think we should print the same message as we would normally print, but prefixed with "[DRY-RUN]"